pkg/sql/alter_role.go

```diff
@@ -65,6 +66,13 @@ const (
 	unknown        setVarBehavior = 3
 )
 
+var changeOwnPasswordEnabled = settings.RegisterBoolSetting(
```

Comment:
Why is this a cluster setting and not a `CHANGEOWNPASSWORD` privilege or something like that?

Reply:
The Product team wanted this to be controlled at the cluster level, and not per-role.